tests/dec/test_dec.py

```diff
@@ -187,14 +185,14 @@ def test_global_photos_flag():
     p = DecFileParser(DIR / "../data/defs-aliases-chargeconj.dec")
     p.parse()
 
-    assert p.global_photos_flag() == True
+    assert p.global_photos_flag()
```

Comment:
Never use `==` with None, True, or False.

Reply:
Ah, did not know that. OK.